templates/assets/schedule.yml

```diff
@@ -0,0 +1,77 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MU scheduled task in a specific environment containing a CloudWatch event and a RuleTarget
+Parameters:
+  EcsServiceRoleArn:
+    Type: String
+    Description: ARN of IAM role for ECS service to assume
+  ServicePort:
```

Comment:
Delete this parameter

Reply:
I need this one because it is referred to by ScheduleRuleName/Properties/Targets:RoleArn.

Reply:
I was referring to `ServicePort`